e2e/web/fixture.ts

```diff
@@ -6,31 +6,36 @@ import { HeaderComponent } from "./components/header.component";
 import { ContactPage } from "./page/contact.page";
 
 import { mixinFixtures as mixinCoverage } from "@bgotink/playwright-coverage";
+import { ALMOST_FINISHED_GAME } from "./data";
```

Comment:
petition to rename this to NEARLY_FINISHED_GAME and add a comment somewhere saying how can a game be nearly finished :)